api/services/ocean/spark/schemas/oceanSparkClusterState.yaml

```diff
@@ -0,0 +1,10 @@
+description: The current state of the Ocean Spark cluster
```

Comment:
Likewise, we need to describe these states. We can reuse what we put in the docs. 
Here's what we put in the regular documentation (google doc with James):

>Progressing - The cluster resources are being created.
Deleting - The cluster resources are being deleted.
Available - The cluster is available.
Unreachable - The cluster stopped sending heartbeat. This can be caused by the cluster going down or by a networking issue between the cluster and Spot's backend.
Degraded - One of the cluster components is unhealthy. Some features may not be available.
Failing - A critical cluster component is unhealthy.
Unknown - The cluster status API has an unexpected, internal error.

cc @thorsteinnth

Reply:
Thanks!